data/kamailio.cfg

```diff
@@ -586,17 +770,29 @@ route[CHECK_ALARM]
 	
 				sht_iterator_start("i1", "h");
 				while(sht_iterator_next("i1")) {
-					if ($shtitval(i1) > $var(avalue)) {
+					if ($shtitval(i1) >= $var(avalue)) {
 						$var(count) = $var(count) + 1;
+						$var(ip) = $(shtitkey(i1){s.select,0,=});
+						$var(ua) = $(shtitkey(i1){s.select,1,=});
 						jansson_set("string", "host", "$var(host)", "$var(tmp_json)");
 						jansson_set("string", "timestamp", "$var(isotime)", "$var(tmp_json)");
-						jansson_set("string", "source_ip", "$shtitkey(i1)", "$var(tmp_json)");
+						jansson_set("string", "source_ip", "$var(ip)", "$var(tmp_json)");
+						jansson_set("string", "useragent", "$var(ua)", "$var(tmp_json)");
 						jansson_set("string", "type", "scanner", "$var(tmp_json)");
 						jansson_set("integer", "value", "$shtitval(i1)", "$var(tmp_json)");
 
 						$var(alarm_data) = $var(alarm_data) + " {\"index\": {\"_index\":\"homer-"+ $timef(%d.%m.%Y) +"\",\"_type\":\""+$var(type)+"\" }}\n";
 						$var(alarm_data) = $var(alarm_data) + $var(tmp_json) + "\n";
 
+						
+						#!ifdef TO_GRAYLOG
+						$var(short) = @cfg_get.short.message;
+						$var(full) = $sel(msg);
+						jansson_set("string", "short_message", ""+$var(short)+"", "$var(tmp_json)");
+						jansson_set("string", "full_message", ""+$var(full)+"", "$var(tmp_json)");
+						$var(graylog_json) = $var(tmp_json);
```

Comment:
This part looks to me repeated in the script. Wouldn't it be better to put it in the SEND_ES_DATA route inside the 
#!ifdef TO_GRAYLOG?
So, using jansson_set on the $var(body).